apps/studio/components/interfaces/Settings/Database/BannedIPs.tsx

```diff
@@ -60,6 +60,16 @@ const BannedIPs = () => {
     setShowUnban(true)
   }
 
+  const [userIPAddress, setUserIPAddress] = useState<string | null>(null)
+
+  useEffect(() => {
+    // Fetch user's IP address
+    fetch(`/api/database/${ref}/get-ip-address`)
```

Comment:
Getting the IP-address of a user is not specific for a project, so there's no need to use that path
```suggestion
    fetch(`/api/get-ip-address`)
```